connector-proxies/async-http/main.py

```
@@ -0,0 +1,132 @@
+import logging
+
+import falcon.asgi
+import httpx
+import orjson
+
+# TODO: change this for prod
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+http_client = httpx.AsyncClient(timeout=None)
```

Ruff (0.12.2)

Probable use of `httpx` call with timeout set to `None`

(S113)